hl7apy/mllp.py

```diff
@@ -47,50 +54,54 @@ def __str__(self):
 
 
 class _MLLPRequestHandler(StreamRequestHandler):
+    encoding = 'utf-8'
```

Comment:
Ok for encoding in utf-8. We plan to let the user specify the encoding he/she wants to use but for now we can use utf-8